libcontainer/cgroups/utils.go

```diff
@@ -277,6 +277,7 @@ func RemovePaths(paths map[string]string) (err error) {
 			}
 		}
 		if len(paths) == 0 {
+			//nolint:ineffassign // done to help garbage collecting: opencontainers/runc#2506
```

Comment:
I don't remember if it's the recommended practice but the comment inside the commend looks a bit weird...

Reply:
Apparently it is... https://golangci-lint.run/usage/false-positives/

Reply:
Yes, it looks a bit weird